dockerizeit/master/matrixbasedsecurity.groovy

```diff
@@ -0,0 +1,30 @@
+import java.util.Properties
+import java.lang.System
+import jenkins.model.*
+import hudson.security.*
+import org.jenkinsci.plugins.*
+import hudson.model.*
+
+println "--> Matrix-Based security: Read properties from the file"
```

Comment:
This print should be under if section. There is no point in printing that if you will not configure security